res/sh/lint.sh

```
@@ -0,0 +1,18 @@
+CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$CWD"/..
+
+set -e
+
+PY_FILES=$(find . -name "*.py")
+
+python3 -m black phlop tests
+pylint --errors-only phlop tests
+isort phlop tests
+
+for FILE in ${PY_FILES[@]}; do
```

Shellcheck

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)